pkg/placement/workload-projector.go

```diff
@@ -1257,7 +1240,7 @@ func (wp *workloadProjector) customizeOrCopy(logger klog.Logger, srcCluster logi
 	}
 	var location *edgeapi.Location
 	if expandParameters {
-		location, err = wp.locationClusterLister.Cluster(logicalcluster.Name(destSP.Cluster)).Get(destSP.LocationName)
+		location, err = wp.edgeClusterClientset.Cluster(logicalcluster.NewPath(destSP.Cluster)).EdgeV2alpha1().Locations().Get(wp.ctx, destSP.LocationName, metav1.GetOptions{})
```

Comment:
Why was this change necessary?

Reply:
I mean, why not fetch the Location object from the local cache of an informer watching Location objects in the KCS?